notebooks/official/custom/sdk-custom-image-classification-batch.ipynb

```diff
@@ -128,182 +131,124 @@
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "013daf3de88e"
-      },
-      "source": [
-        "## Before you begin"
+        "! pip3 install --upgrade google-cloud-aiplatform \\\n",
+        "                        google-cloud-storage \\\n",
+        "                        pillow  \\\n",
+        "                        numpy    "
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "8bc8a29f9001"
+        "id": "ff555b32bab8"
       },
       "source": [
-        "#### Set your project ID\n",
+        "### Restart runtime (Colab only)\n",
         "\n",
-        "**If you don't know your project ID**, try the following:\n",
-        "* Run `gcloud config list`.\n",
-        "* Run `gcloud projects list`.\n",
-        "* See the support page: [Locate the project ID](https://support.google.com/googleapi/answer/7014113)"
+        "To use the newly installed packages, you must restart the runtime on Google Colab."
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "e61aaa036444"
+        "id": "f09b4dff629a"
       },
       "outputs": [],
       "source": [
-        "PROJECT_ID = \"[your-project-id]\"  # @param {type:\"string\"}\n",
+        "import sys\n",
         "\n",
-        "# Set the project id\n",
-        "! gcloud config set project {PROJECT_ID}"
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "c4a624c8099d"
-      },
-      "source": [
-        "#### Region\n",
+        "if \"google.colab\" in sys.modules:\n",
         "\n",
-        "You can also change the `REGION` variable used by Vertex AI. Learn more about [Vertex AI regions](https://cloud.google.com/vertex-ai/docs/general/locations)."
-      ]
-    },
-    {
-      "cell_type": "code",
-      "execution_count": null,
-      "metadata": {
-        "id": "f83bd6013894"
-      },
-      "outputs": [],
-      "source": [
-        "REGION = \"us-central1\"  # @param {type: \"string\"}"
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "08bfd1eb44ef"
-      },
-      "source": [
-        "### Authenticate your Google Cloud account\n",
+        "    import IPython\n",
         "\n",
-        "Depending on your Jupyter environment, you may have to manually authenticate. Follow the relevant instructions below."
+        "    app = IPython.Application.instance()\n",
+        "    app.kernel.do_shutdown(True)"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "af349043f23b"
+        "id": "54c5ef8a8f43"
       },
       "source": [
-        "**1. Vertex AI Workbench**\n",
-        "* Do nothing as you are already authenticated."
+        "<div class=\"alert alert-block alert-warning\">\n",
+        "<b>⚠️ The kernel is going to restart. Please wait until it is finished before continuing to the next step. ⚠️</b>\n",
```

Comment:
Remove "Please" and contract "it is":

- Wait until it's finished before...

Reply:
Did required changes